control/narrative_table.go

```diff
@@ -0,0 +1,91 @@
+package control
+
+import (
+	"errors"
+	"regexp"
+
+	"github.com/jbowtie/gokogiri/xml"
+	"github.com/opencontrol/fedramp-templater/docx/helper"
+	"github.com/opencontrol/fedramp-templater/opencontrols"
+)
+
+func findSectionKey(row xml.Node) (section string, err error) {
+	re := regexp.MustCompile(`Part ([a-z])`)
```

Comment:
The question for FedRamp is what would they do if the Part ever went beyond `z`. (it doesn't get close now. stops around part k)